website/templates/institution.mako

```diff
@@ -11,7 +11,7 @@
         <div class="dashboard-header dashboard-header-institution">
             <div class="row" style="text-align: center">
                 % if banner_path:
-                    <div class="col-sm-6 col-sm-offset-3"><img height="110px" src="${ banner_path }"></div>
+                    <div class="col-sm-6 col-sm-offset-3"><img style="max-height: 100%; max-width:100%;" src="${ banner_path }"></div>
```

Comment:
For a11y, all images should have an `alt` attribute. Can you easily add the institution's name as the `alt` value?